metabase_api/metabase_api.py

```diff
@@ -778,8 +778,64 @@ def copy_collection(self, source_collection_name=None, source_collection_id=None
         self.copy_pulse(source_pulse_id=pulse_id,
                         destination_collection_id=destination_parent_collection_id,
                         destination_pulse_name=destination_pulse_name)
-  
-  
+
+
+  def search(self, q, item_type=None, archived=False, single_ret=False):
+    """
+    Search content and filter item type. Note: endpoint /api/search fall behind with web app version.
+
```

Comment:
One last question: What do you mean by "endpoint /api/search fall behind with web app version"?

Reply:
Metabase web API provides filter content type param, such as 'card' type, whereas endpoint /api/search does not support this filter param.
**Web API:** 
![image](https://user-images.githubusercontent.com/5664723/116370658-31fbf780-a83d-11eb-8dc0-6a64fd428623.png)
**API in their docs:**
![image](https://user-images.githubusercontent.com/5664723/116371008-8b642680-a83d-11eb-9970-53e84c50fcb8.png)

Reply:
Where it says `&type=card` if you look at the `response` tab, you see that the response contains all types. So `type` is apparently something used for the frontend. Under the hood all the results have actually been returned.